### PR TITLE
Ogs in container

### PR DESCRIPTION
**Pull request by jbathmann**
As discussed, this PR provides functionality to run ogs in a container.

It should work out of the box. However, the container needs to be downloaded and singularity needs to be installed. Instructions should be found trying to use this PR. @zaphod-bx @jvollhueter please test.

Two test setups are provided and can be run from the MANGA main directory using

- python3 MANGA.py -i test/LargeTests/Test_Setups_large/ogs_example_setup/OGS3D_SAZOI_BETTINA.xml
- python3 MANGA.py -i test/LargeTests/Test_Setups_large/example_box_amp_0_5_kappa_1e-11_slope_0_001/setup_pymanga.xml

### CODE REVIEW

**Comment by jbathmann** [comment]
I updated the readme.txt file after a discussion with @zaphod-bx . Thanks for the comments!

Please feel free for further reviews.

**Comment by jbathmann** [comment]
@mcwimm 
Rebase after merge of #157

File: TreeModelLib/BelowgroundCompetition/OGSLargeScale3D/OGSLargeScale3D.py [hunk 1]
```diff
@@ -317,21 +320,26 @@ def copyPythonScript(self):
 
             # OGS
             if "constant_contributions.npy" in line:
-                line = line.replace("constant_contributions.npy",
-                                    constants_filename)
+                line = line.replace(line,
+                                    "constant_contributions = np.load('" +
```
Comment (mcwimm) [inline TreeModelLib/BelowgroundCompetition/OGSLargeScale3D/OGSLargeScale3D.py:324]: For it to run with Windows, all paths need to be interpreted as raw string -> add `r` before each path.

```suggestion
                                    "constant_contributions = np.load(r'" +
```

**Review by mcwimm — COMMENTED** [review]
With the proposed change both test-scripts work for me (i.e. using ogs installed locally not as container).

File: TreeModelLib/BelowgroundCompetition/OGSLargeScale3D/OGSLargeScale3D.py (hunk 1 above)
Comment (mcwimm) [reply TreeModelLib/BelowgroundCompetition/OGSLargeScale3D/OGSLargeScale3D.py]: see commit... does it still work on linux?

Comment (jbathmann) [reply TreeModelLib/BelowgroundCompetition/OGSLargeScale3D/OGSLargeScale3D.py]: Works perfectly fine.
Thanks for the commit!
I tried to update #158 accordingly. Can you please test?

**Comment by mcwimm** [comment]
imo it's ready to be merged

**Comment by jbathmann** [comment]
Thanks @mcwimm .
I'd like to wait for @jvollhueter to have a look on this before we merge.

**Comment by jvollhueter** [comment]
I could not start the container with:

Ubuntu 20.04 on a VM (VirtualBox 6.1.36), Python 3.8 and
Ubuntu 22.04 on a VM (VirtualBox 6.1.36), Python 3.10.

Further I could not create the container on Ubuntu 20.04 by myself (tried twice, resulted in breaking the whole OS).

**Pull request merged**